src/main/resources/data/ancientology/recipes/bone_axe.json

```diff
@@ -0,0 +1,18 @@
+{
+	"type": "minecraft:crafting_shaped",
```

Comment:
The formatting in this file is very weird.

Reply:
How so? It is exactly how Turty does it in his tutorials